env/dev.json

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
@oscarmild 
**Is this file for dev server?**

Reply:
@fromgopi 
Yes this is for dev server.

Reply:
@oscarmild 
Is dev server up?

Reply:
@fromgopi No server team is in the process of setting up the dev server

Reply:
@oscarmild Ok but make sure that is done by the EoD so that we can push some code into dev server and see how it works. Also make sure that all the sql stuff are executed before the server is jump started. If this happens we can release the v0.0.1 version for the board and ask them to check. We will also make sure that the scope of the project will be limited to the tasks and how PM will manage it.